src/bundle/Resources/public/js/scripts/admin.notifications.list.js

```diff
@@ -22,6 +24,7 @@
             .then(getJsonFromResponse)
             .then((response) => {
                 if (response.status === 'success') {
+                    if (isFirefox) clearCheckboxes();
```

Comment:
```suggestion
                    if (isFirefox()) {
                        clearCheckboxes();
                    }
```
I think isFirefox is a function, and for consistency of the code let's not use one line if.